specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -1824,6 +1982,259 @@
+          },
+          "description": "The alert rule template status."
+        },
+        "requiredDataConnectors": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DataConnectorStatus"
+          },
+          "description": "The required data connectors for this template"
+        },
+        "alertRulesCreatedByTemplateCount": {
+          "type": "integer",
+          "description": "the number of alert rules that was created by this template"
+        }
+      }
+    },
+    "ScheduledAlertRuleTemplateProperties": {
+      "type": "object",
+      "description": "Schedule alert rule template property bag.",
+      "properties": {
+        "severity": {
+          "$ref": "#/definitions/AlertRuleSeverity",
+          "description": "The severity for alerts created by this alert rule."
+        },
+        "query": {
+          "type": "string",
+          "description": "The query that creates alerts for this rule."
+        },
+        "queryFrequency": {
+          "type": "string",
+          "format": "duration",
+          "description": "The frequency (in ISO 8601 duration format) for this alert rule to run."
+        },
+        "queryPeriod": {
+          "type": "string",
+          "format": "duration",
+          "description": "The period (in ISO 8601 duration format) that this alert rule looks at."
+        },
+        "triggerOperator": {
+          "$ref": "#/definitions/AlertRuleTriggerOperator",
+          "description": "The operation against the threshold that triggers alert rule."
+        },
+        "triggerThreshold": {
+          "type": "integer",
+          "description": "The threshold triggers this alert rule."
+        }
+      }
+    },
+    "FilterAlertRuleTemplate": {
+      "x-ms-discriminator-value": "Filter",
+      "type": "object",
+      "description": "Represents filter alert rule template.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AlertRuleTemplate"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Filter alert rule template properties",
+          "allOf": [
+            {
+              "$ref": "#/definitions/BaseAlertRuleTemplateProperties"
+            },
+            {
+              "$ref": "#/definitions/FilterAlertRuleTemplateProperties"
+            }
+          ],
+          "required": [
+            "displayName",
+            "description",
+            "createdDateUTC",
+            "status",
+            "alertRulesCreatedByTemplateCount",
+            "filterProduct"
+          ]
+        }
+      }
+    },
+    "FilterAlertRuleTemplateProperties": {
+      "type": "object",
+      "description": "Filter alert rule template property bag.",
+      "properties": {
+        "filterProduct": {
+          "type": "string",
+          "description": "The filter prodact name for this template rule."
+        },
+        "filterSeverities": {
```

Comment:
Will there be templates with builtin severities and titles?

Reply:
In the future